internal/api/http/firecracker/v1/firecracker.go

```diff
@@ -10,6 +10,7 @@ type Drive struct {
 	PathOnHost   string `json:"path_on_host"`
 	IsRootDevice bool   `json:"is_root_device"`
 	IsReadOnly   bool   `json:"is_read_only"`
+	IOEngine     string `json:"io_engine"`
```

Comment:
This should be an enum, not a string

Reply:
Resolved in fba812e610826632ed649be89d0d01df7d7b073f